lib/dice_roller_ui.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'utils/constants.dart';
+import 'utils/text_styles.dart';
+
+class DiceRollerUI extends StatefulWidget {
+  DiceRollerUI({
+    Key key,
+    this.title,
+  }) : super(key: key);
+
+  final String title;
+
+  @override
+  _DiceRollerUIState createState() => _DiceRollerUIState();
+}
+
+class _DiceRollerUIState extends State<DiceRollerUI> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          widget.title,
+        ),
+      ),
+      backgroundColor: Color.fromARGB(
+        Constants.backgroundColor255,
+        Constants.backgroundColor0,
+        Constants.backgroundColor217,
+        Constants.backgroundColor255,
```

Comment:
Try to not add to the name of the constant the value that is going to have. In this case you can change this names with: backgroundColorAlpha, backgroundColorRed, backgroundColorGreen, backgroundColorBlue